### DISCUSSION TITLE
Sepetteki elmalar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#300] [Zeka Sorusu] [answered]
Bulent, sepetteki elmaların yarısını ve bir yarım elmayı Ayşegül ’e; sonra kalan elmaların yarısını ve bir yarım elmayı Evren'e ve yine kalan elmaların yarısını ve bir yarım elmayı da Melike ’ye veriyor.
Sonuçta sepette sadece bir elma kaldığına göre başlangıçta kaç elma vardı?

**Comment by adaakoluk** [comment, marked as answer]
Cevap 15. Ters işlem yaparak gittiğimizde:
1 + 0,5
1,5 x 2
3,0 + 0,5
3,5 x 2 
7,0 + 0,5
7,5 x 2
15
